libass/ass_types.h

```diff
@@ -229,6 +229,12 @@ typedef struct ass_event {
  * absolutely can't do that, because the video colorspace is required
  * in order to handle this as intended. API users must use the exposed
  * information to perform color mangling as described above.
+ *
+ * Further note all of the above only concerns the RGB values.
+ * Gamut mapping and transfer charateristics of ASS subtitles
```

Comment:
As far as I'm aware, "Gamut mapping" specifically refers to the process of mapping colors in one gamut to another gamut (as a color analog of tonemapping), rather than the gamut any given RGB triple is associated with. So maybe "Color primaries" would fit better here.

Reply:
Indeed “color primaries” seems more appropiate here afaict, thanks!